ribs/visualize.py

```diff
@@ -290,20 +314,25 @@ def cvt_archive_heatmap(archive,
         cmap (str, list, matplotlib.colors.Colormap): Colormap to use when
             plotting intensity. Either the name of a colormap, a list of RGB or
             RGBA colors (i.e. an Nx3 or Nx4 array), or a colormap object.
-        square (bool): If True, set the axes aspect ratio to be "equal".
+        square (bool): [DEPRECATED]
+        aspect ('auto', 'equal', float) [optional]: the aspect ratio of the heatmap. Defaults to 'auto' for 2D. 'equal' is the same as ``aspect=1``.
         ms (float): Marker size for both centroids and samples.
         lw (float): Line width when plotting the voronoi diagram.
         vmin (float): Minimum objective value to use in the plot. If None, the
             minimum objective value in the archive is used.
         vmax (float): Maximum objective value to use in the plot. If None, the
             maximum objective value in the archive is used.
+        cbar (str, matplotlib.axes.Axes): By default, this is set to 'auto' which displays the colorbar on the archive's current Axes. If None, then colorbar is not displayed. If this is an Axes object, displays the colorbar on the specified Axes
+        cbar_kwargs (dict): Additional kwargs to pass to :func:`~matplotlib.figure.Figure.colorbar`
+
     Raises:
         ValueError: The archive is not 2D.
     """
-    # pylint: disable = too-many-locals
+    if aspect is None:
+        aspect = "auto"
 
-    if archive.behavior_dim != 2:
-        raise ValueError("Cannot plot heatmap for non-2D archive.")
+    _validate_heatmap_visual_args(aspect, cbar, square, archive.behavior_dim,
+                                  [2], "CVT archive must be 2D")
```

Comment:
For the error, specify that this is a heatmap-specific error, i.e. say
`"Heatmaps can only be plotted for 2D CVTArchive"` -- also do the same for the GridArchive error message.